Makefile

```diff
@@ -27,57 +30,68 @@ clean: ## Clean build directory
 	cd $(DOCS_DIR) && rm -rf $(BUILDDIR)/*
 
 .PHONY: build
-build:  ## Set up training: Install requirements
+build:		## Set up training: Install requirements
 	python3 -m venv . || virtualenv --clear --python=python3 .
 	bin/python -m pip install --upgrade pip
 	bin/pip install -r requirements.txt
-	@echo
-	@echo "Please activate your Python virtual environment with"
-	@echo "source bin/activate"
+
+bin/python:
+	python3 -m venv . || virtualenv --clear --python=python3 .; \
+	bin/python -m pip install --upgrade pip; \
+	bin/pip install -r requirements.txt
+
+docs/volto:
+	# git submodule add git@github.com:plone/volto.git submodules/volto
```

Comment:
What is the purpose of this comment?

Reply:
I can remove. Just as a reminder of how the submodule was added.

Reply:
That is what I assumed. Let's get it documented in `docs/contributing/authors.md`, as it is one of those things that is hard to remember, and we might need to do this for other repos (maybe restapi?). I've added this as a task

Reply:
@tiberiuichim please check the commit 649811f0d0ddabf7f918e234c6d76e5fc37415a0 to make sure I understood how to do this.

Reply:
Resolving per subsequent reviews from @ksuess and @nileshgulia1